lang/lang_en.txt

```diff
@@ -382,6 +385,9 @@
 #MSG_LAST_PRINT_FAILURES c=20
 "Last print failures"
 
+#
```

Comment:
should be at least `# c=20 r=7` better `#MSG_WIZARD_WELCOME_SHIPPING c=20 r=7` if moved to `messages.*`